### PR TITLE
Add "raw request gRPCurl" output

### PR DESCRIPTION
**Pull request by dariusdamalakas**
Implements [#3 Copy as "grpcurl"](https://github.com/fullstorydev/grpcui/issues/3). 

**Example:**
image

**With highlights of the changes parts:**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,12 @@ run `make install`.
 If you encounter compile errors, you could have out-dated versions of `grpcui`'s
 dependencies. You can update the dependencies by running `make updatedeps`.
 
+### Running without install 
+
+```
+go run ./cmd/grpcui/grpcui.go -plaintext localhost:9019
```
Comment (dariusdamalakas) [inline README.md:82]: It was not clear for me how to run the app locally, took me 10 minutes or so to figure out.

File: internal/resources/webform/webform-template.html [hunk 2]
```diff
@@ -46,7 +46,7 @@ <h3>Examples</h3>
     <div id="grpc-request-response">
       <ul>
           <li id="#grpc-request-tab-button"><a href="#grpc-request-tab">Request Form</a></li>
-          <li id="#grpc-request-raw-tab-button"><a href="#grpc-request-raw-tab">Raw Request (JSON)</a></li>
+          <li id="#grpc-request-raw-tab-button"><a href="#grpc-request-raw-tab">Raw Request</a></li>
```
Comment (dariusdamalakas) [inline internal/resources/webform/webform-template.html:49]: Removing `(JSON)` as this is not adding any useful information

File: internal/resources/webform/webform-template.html [hunk 3]
```diff
@@ -132,6 +137,7 @@ <h3>Response Trailers</h3>
         {"name": "{{.Name}}", "value": "{{.Value}}"},
     {{- end}}
     ];
+    window.target = {{ .Target }};
```
Comment (dariusdamalakas) [inline internal/resources/webform/webform-template.html:140]: `Target` is the address of the grpcServer where `grpcui` is connecting to.

File: internal/resources/webform/webform-template.html [hunk 4]
```diff
@@ -70,7 +70,12 @@ <h3>Request Timeout</h3>
         <button class="grpc-invoke" disabled>Invoke</button>
       </div>
       <div class="grpc-tabcontent" id="grpc-request-raw-tab">
+        <h3>Request payload</h3>
         <textarea id="grpc-request-raw-text"></textarea><br/>
+        <h3>gRPC curl</h3>
+        <div id="grpc-curl" class="grpc-curl-panel">
+          <span><pre id="grpc-curl-text"></pre></span>
+        </div>
```
Comment (dragonsinth) [inline internal/resources/webform/webform-template.html:78]: This PR is awesome, thank you.  My one question is: why the different constructions (textarea vs. pre)?
Seems like we should use the same setup and formatting for both.  I'm not strongly opinionated on which... if you like the new formatting, you could just make the CSS classname something like `grpc-request-panel` and apply it to both.

Comment (dragonsinth) [reply internal/resources/webform/webform-template.html:78]: Holy crap... I actually had no idea that the Raw Request box was even editable.....

Comment (dragonsinth) [reply internal/resources/webform/webform-template.html:78]: In that case, I wonder if we can move the `Invoke` button up above the grpcurl area?

I also wonder if there's a way to make it more obvious that the request box is editable..

Comment (dariusdamalakas) [reply internal/resources/webform/webform-template.html:78]: @dragonsinth    Like this?  

I am not a fan of this, looks messy. Buttons are usually at the bottom. 


image

Comment (dragonsinth) [reply internal/resources/webform/webform-template.html:78]: Seems fine to me.  I think it's weirder to separate the editable field from the button that it goes with.

Comment (dariusdamalakas) [reply internal/resources/webform/webform-template.html:83]: @dragonsinth pushed new changes


image

File: internal/resources/webform/webform-sample.css [hunk 5]
```diff
@@ -745,3 +745,15 @@ button#grpc-history-clear, button#grpc-history-save {
 #grpc-history-list .history-item-panel .history-detail-metadata td {
     padding-left: 12px;
 }
+
+.grpc-request-raw-container {
+    margin-bottom: 1em;
+}
+
+
```
Comment (dragonsinth) [inline internal/resources/webform/webform-sample.css:753]: nit: extra blank line here

**Comment by dariusdamalakas** [comment]
@dragonsinth thanks.  Removed empty line!

File: internal/resources/webform/webform-sample.css (hunk 5 above)
Comment (dariusdamalakas) [reply internal/resources/webform/webform-sample.css]: Removed. Thanks!

**Comment by jhump** [comment]
First nit: it's called "grpcurl" or "[gRPCurl](https://github.com/fullstorydev/grpcurl#grpcurl)", not "gRPC curl".

Another few cents worth of UX feedback, take it or leave it:

This seems overly prominent in the UI... a bit of an eyesore and possible even confusing. Should users try to paste existing `grpcurl` invocations into the textarea, like they can with JSON? Or is it read-only? While it's possibly interesting information, it is more likely a distraction for most people actually using the "Raw Request" tab.

Following how Chrome exposes similar functionality in its dev tools, I think it would be better to instead add a "Copy as grpcurl" button to each row in the "History" tab.

**Comment by dragonsinth** [comment]
This seems like a reasonable incremental thing.  If some sufficiently motivated person wants to turn this into "copy as gRPCurl" that might be a nice improvment.

**Comment by dragonsinth** [comment]
FYI: I pushed a commit to do 2 things:

1) Adjust the spacing a tad, break up with an HR
2) (javascript) ensure consistency between editing the form and copying as curl by shuffling the code a tad.

**Comment by dragonsinth** [comment]
Screenshot 2023-04-27 at 6 18 17 PM

**Pull request merged**